packages/core/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -179,6 +181,10 @@ ChoiceList.propTypes = {
    */
   hint: PropTypes.node,
   /**
+   * Text showing the requirement ("Required", "Optional", etc.). See []().
```

Comment:
TODO: Add the link (or remove the "See..." part)

Reply:
I replaced it with a link that should work, but it is replacing `{{root}}` with the wrong value for some reason. I've looked at the responsible functions, but I don't see any apparent issues.